### ISSUE TITLE
Bug: renaming a thread says that a new version has been created in the discussion area

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#3293]
## Bug description & symptom

Renaming a thread says that a new version has been created in the discussion area. The "new version" element should instead say that the discussion thread has been renamed.

I actually think that this should be done for the other apps too.

@celiamgtt?

## Impacted components
- *frontend*

**Comment by celiamgtt** [comment]
Yes it could be interesting to be more specific with the 'new version' message in the timeline
